### DISCUSSION TITLE
The PostgreSQL container runs in duplicate with spring-boot-test

### DISCUSSION THREAD

**Discussion by kdevkr** [testcontainers/testcontainers-java#6398] [Q&A] [answered]
The PostgreSQL container runs in duplicate. It seems that the container shared by the test code is not used.

https://github.com/kdevkr/spring-demo-testcontainers/pull/8

The container by PostgreSQLContainer and the container by designation jdbc:tc:postgresql:14.6 run simultaneously. 
I want to use the first configuration in the spring boot test.

This results in the parameter option not being applied as intended.

Please give me a hint about the wrong part. 😥

**Comment by eddumelendez** [comment, marked as answer]
I have left a comment on the PR
